comms/discovery/rpcz/apply.go

```diff
@@ -199,8 +199,19 @@ func Apply(msg *nats.Msg) {
 			if err != nil {
 				config.Logger.Warn("failed to load chat members for websocket push " + err.Error())
 			} else {
-				for _, userId := range userIds {
-					websocketPush(userId, msg.Data)
+				var parsedParams schema.RPCPayloadParams
+				err := json.Unmarshal(rawRpc.Params, &parsedParams)
+				if err != nil {
+					config.Logger.Error("Failed to parse params")
+				}
+				payload := schema.RPCPayload{ Method: schema.RPCMethod(rawRpc.Method), Params: parsedParams }
+				encodedUserId, err := misc.EncodeHashId(int(userId))
+				data := schema.ChatWebsocketEventData{ RPC: payload, Metadata: schema.Metadata{Timestamp: meta.Timestamp.Format(time.RFC3339Nano), UserID: encodedUserId}}
+				for _, subscribedUserId := range userIds {
+					// Don't send events sent by a user to that same user
```

Comment:
there's a ticket to make the mutate endpoint block until local apply succeeds...
but sending the websocket event back to current user could also be helpful in that context while we await that ticket.

Reply:
unf the SDK doesn't have context who they are atm 😅 so it's impossible to differentiate within the SDK. However, the client middleware could have context. It would couple the middleware impl with the store state, though, which I'm not super keen on, and I think loading that state might be async so there's a possibility of getting a chat before the accountUserId is set. Would have to investigate more. But ultimately, it's probably more important to know that a request didn't get relayed than it is to know it did, which I don't think the websocket helps with here